### DISCUSSION TITLE
Approaches to shipping

### DISCUSSION THREAD

**Discussion by janbaykara** [commonknowledge/leftbookclub#59] [Project Decisions] [answered]
We've discussed how to manage shipping fees for products, but I wanted to offer a simplifcation that we can possibly do, that comes from working on book purchases (#4):

### Status quo approach: Stick with per-country subscription prices
- This works as described over email and in the [user wiki](https://github.com/commonknowledge/leftbookclub/wiki#which-prices-are-used)
- Is a bit fussy to manage these prices in Stripe, perhaps. Product prices need to be calculated to include shipping, and metadata needs to be applied to those prices with the correct shipping zone, whose codes are defined back in Wagtail.
- Shipping doesn't show up as a separate line item in checkout, invoices and in the billing portal for customers.

### Alternative approach: Use line items
- Country-specific prices don't need to be managed in Stripe. Instead, we just set the shipping fees for each Shipping Zone (e.g. UK, EU, ROW) in wagtail
- Shipping fees show up as a separate line item (see screenshots below)
- Downside: can't do "Upgrade to Annual Payments" in Checkout. As a mitigation, we could add a "yearly" option to the plan picker section.

#### How this would look in Checkout

<img width="1055" alt="Screenshot 2022-04-13 at 17 32 36" src="https://user-images.githubusercontent.com/237556/163228308-db59c9b8-b7bb-4bd6-8c31-5616eb5f330c.png">

#### How this would look in the Customer Billing Portal

<img width="629" alt="Screenshot 2022-04-13 at 16 59 48" src="https://user-images.githubusercontent.com/237556/163228298-7762a2d0-5b2e-416e-89df-4c6d06cf9f2a.png">

**Comment by janbaykara** [comment]
What do you think about the alternative approach @melaniepatrick?

**Comment by melaniepatrick** [comment, marked as answer]
Hi
Thanks. I like the idea of it being simpler to manage. And being able to change the prices of either shipping or subs easily. So if the alternative idea gives us more flexibility on setting and managing prices that's good?

I like your alternative suggestion because it's important that shipping shows up separately in the checkout. But I'd like to walk through the options in practice.

Upselling to annual: pity to lose this option but it may be less important than the shipping. I think we need to think carefully about how to encourage people to sign up for annual subs.
